docs/conf.py

```diff
@@ -89,6 +89,9 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
 
+def setup(app):
+    app.add_js_file("js/matomo.js")
```

Comment:
Looks like maybe you forgot to add the actual file?

Reply:
Uh, yeah, and I also seem to have added a bunch of unrelated changes. Looks screwed up. Gonna fix it in a minute.

Reply:
Ok, PR should be fixed.